build.sbt

```diff
@@ -22,35 +22,34 @@ lazy val buildSettings = Seq(
   bintrayPackage := s"${organization.value}:${name.value}_${scalaBinaryVersion.value}"
 )
 
-lazy val testkit = Project(
-  id = "testkit",
-  base = file("testkit"),
-  settings = buildSettings
-).enablePlugins(LibraryPlugin)
+lazy val caching = Project(id = "caching", base = file("caching"))
```

Comment:
Nitpick: The subproject name should match the package name - either both should be `cache`, or both should be `caching`.